### ISSUE TITLE
Value List (Advanced Panel) - Avoid improper nesting of form elements and links - (2036570518)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5635]
### Module: 
 42 Calcite Panel 

### Violation: 
 Avoid improper nesting of form elements and links 

![image](https://user-images.githubusercontent.com/5023024/198110188-50c505c6-1bb4-411e-841a-0b3b1e1f7742.png)

### WCAG Reference: 
- [4.1.1 Parsing](https://www.w3.org/WAI/WCAG21/Understanding/parsing.html) 

### Instance ID: 
 2036570518 

### Severity: 
 7 

### Description: 
 [Issue]
There are show/hide setting image controls without appropriate state information.

[User Impact]
Screen reader users will be unable to determine that these controls expand and collapse content. They will also have difficulty determining whether a control is currently expanded or collapsed.

[Code Reference] 
 ```
<button aria-busy="false" aria-disabled="false" aria-label="Settings for events" class="button">
  (...)
</button>
 ``` 

### Note: 
 [Suggestion]
Ensure custom controls provide proper textual name, role, and state information.

For show/hide controls, the following information is expected:
- The element that shows and hides a content area must be a ``` <button> ```.
- When the content area is visible, the button must have aria-expanded="true". Otherwise, the button must have aria-expanded="false".
- The button must have an accessible name set with internal text, aria-label, or aria-labelledby.

[Compliant Code Example] 
 ```
<button aria-expanded="true" aria-busy="false" aria-disabled="false" aria-label="Settings for events" class="button">
  (...)
</button>
 ```
 

### Media Type: 
 Forms 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
While the error surfaces in panel, is related to the `calcite-value-list` component per the [sample](https://geospatialem.github.io/calcite-samples/1-panel.html), and could be a result of the configuration (_see:_ [html code](https://github.com/geospatialem/calcite-samples/blob/main/1-panel.html)).

![image](https://user-images.githubusercontent.com/5023024/198110188-50c505c6-1bb4-411e-841a-0b3b1e1f7742.png)

**Comment by driskull** [comment]
I think this would be a user configuration solution

**Comment by geospatialem** [comment]
Agreed - closing this issue.
